erpnext/patches/v11_1/delete_bom_browser.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+    frappe.delete_doc('Page', 'bom-browser')
```

Comment:
Use `frappe.delete_doc_if_exists`, otherwise it will fail for new instance where the page does not exist.

Reply:
done